lightly/loss/byol_loss.py

```diff
@@ -0,0 +1,189 @@
+import singleton as singleton
+import torch
+from torch import nn
+
+import torch.nn.functional as F
+
+from models.mlp import MLP
+
+
```

Comment:
Please remove `ExperimentParams` from `byol_loss.py`.

If there is the wish for better experiment tracking, open an issue about it and then we can discuss the next steps.